openlr/openlr_match_quality/openlr_assessment_tool/main.cpp

```diff
@@ -6,11 +6,30 @@
 
 #include <QApplication>
 
+namespace
+{
 DEFINE_string(resources_path, "", "Path to resources directory");
 DEFINE_string(data_path, "", "Path to data directory");
+DEFINE_string(login, "", "Login string");
+DEFINE_string(paswd, "", "Password string");
```

Comment:
s/paswd/password/
No need to keep beautiful string length invariant here.

Reply:
Done.